pkg/api/api.go

```diff
@@ -288,7 +288,7 @@ func (hs *HTTPServer) registerRoutes() {
 
 			// Allow HTTP access to the object storage feature (dev only for now)
 			if hs.Features.IsEnabled(featuremgmt.FlagGrpcServer) {
-				apiRoute.Group("/object", hs.httpObjectStore.RegisterHTTPRoutes)
+				apiRoute.Group("/object", hs.httpEntityStore.RegisterHTTPRoutes)
```

Comment:
should this be `/entity`?

Reply:
yup -- I am sure there are more 😬

Reply:
I will merge this as is, and create a new PR that has a few more changes with less automatic renamings

Reply:
good call. it's pretty hard to spot in this one